build.go

```diff
@@ -80,25 +93,24 @@ type TestResult struct {
 	SkipCount int64   `json:"skipCount"`
 	Suites    []struct {
 		Cases []struct {
-			Age             int64       `json:"age"`
-			ClassName       string      `json:"className"`
-			Duration        float64     `json:"duration"`
-			ErrorDetails    interface{} `json:"errorDetails"`
-			ErrorStackTrace interface{} `json:"errorStackTrace"`
-			FailedSince     int64       `json:"failedSince"`
-			Name            string      `json:"name"`
-			Skipped         bool        `json:"skipped"`
-			SkippedMessage  interface{} `json:"skippedMessage"`
-			Status          string      `json:"status"`
-			Stderr          interface{} `json:"stderr"`
-			Stdout          interface{} `json:"stdout"`
+			Age             int64   `json:"age"`
+			ClassName       string  `json:"className"`
+			Duration        float64 `json:"duration"`
+			ErrorDetails    string  `json:"errorDetails"`
+			ErrorStackTrace string  `json:"errorStackTrace"`
+			FailedSince     int64   `json:"failedSince"`
+			Name            string  `json:"name"`
+			Skipped         string  `json:"skippedMessage"`
+			Status          string  `json:"status"`
+			Stderr          string  `json:"stderr"`
+			Stdout          string  `json:"stdout"`
 		} `json:"cases"`
-		Duration  float64     `json:"duration"`
-		ID        interface{} `json:"id"`
-		Name      string      `json:"name"`
-		Stderr    interface{} `json:"stderr"`
-		Stdout    interface{} `json:"stdout"`
-		Timestamp interface{} `json:"timestamp"`
+		Duration  float64          `json:"duration"`
+		ID        string           `json:"id"`
+		Name      string           `json:"name"`
+		Stderr    string           `json:"stderr"`
+		Stdout    string           `json:"stdout"`
+		Timestamp RFC1123Timestamp `json:"timestamp"`
```

Comment:
https://jenkins.in.bench.co/view/Masters/job/BenchLabs/job/eventing-service/job/master/232/testReport/api/json

You also want:

```
		EnclosingBlockNames []string    `json:"enclosingBlockNames"`
		EnclosingBlocks     []string    `json:"enclosingBlocks"`
```

```
type AutoGenerated struct {
	Class       string        `json:"_class"`
	TestActions []interface{} `json:"testActions"`
	Duration    float64       `json:"duration"`
	Empty       bool          `json:"empty"`
	FailCount   int           `json:"failCount"`
	PassCount   int           `json:"passCount"`
	SkipCount   int           `json:"skipCount"`
	Suites      []struct {
		Cases []struct {
			TestActions     []interface{} `json:"testActions"`
			Age             int           `json:"age"`
			ClassName       string        `json:"className"`
			Duration        float64       `json:"duration"`
			ErrorDetails    interface{}   `json:"errorDetails"`
			ErrorStackTrace interface{}   `json:"errorStackTrace"`
			FailedSince     int           `json:"failedSince"`
			Name            string        `json:"name"`
			Skipped         bool          `json:"skipped"`
			SkippedMessage  interface{}   `json:"skippedMessage"`
			Status          string        `json:"status"`
			Stderr          string        `json:"stderr"`
			Stdout          string        `json:"stdout"`
		} `json:"cases"`
		Duration            float64     `json:"duration"`
		EnclosingBlockNames []string    `json:"enclosingBlockNames"`
		EnclosingBlocks     []string    `json:"enclosingBlocks"`
		ID                  string      `json:"id"`
		Name                string      `json:"name"`
		NodeID              string      `json:"nodeId"`
		Stderr              string      `json:"stderr"`
		Stdout              string      `json:"stdout"`
		Timestamp           interface{} `json:"timestamp"`
	} `json:"suites"`
}
```